GOVERNANCE.md

```diff
@@ -0,0 +1,98 @@
+# Project Contribution Guidelines
+
+## Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project design
+and participate in the decision making process. This document describes how that
+participation takes place and how to set about earning merit within the project
+community.
+
```

Comment:
Was this Governance document inspired by some existing project's governance document?

If yes, it would be nice to tell about it here so that seasoned contributors have easier understanding and comparison to a governance document of an established project.

Reply:
Good idea, added to the top-level section, please take another look.

Reply:
Looks good, thanks!